index.d.ts

```diff
@@ -471,6 +499,35 @@ declare namespace Moleculer {
 		}
 	}
 
+	class Transit {
+		constructor(broker: ServiceBroker, transporter: Transporter, opts: GenericObject);
+		afterConnect(wasReconnect: boolean): Promise<void>;
+		connect(): Promise<void>;
+		disconnect(): Promise<void>;
+		sendDisconnectPacket(): Promise<void>;
+		makeSubscriptions(): Promise<Array<void>>;
+		messageHandler(cmd: string, msg: GenericObject): boolean | Promise<void> | undefined;
+		request(ctx: Context): Promise<void>;
+		sendEvent(nodeID: string, eventName: string, data: GenericObject): void;
+		sendBalancedEvent(eventName: string, data: GenericObject, nodeGroups: GenericObject): void;
+		sendEventToGroups(eventName: string, data: GenericObject, groups: Array<any>): void;
```

Comment:
I'm sure the `groups` param could have a better definition

Reply:
It's `Array<String>` or null/undefined